java/src/org/openqa/selenium/remote/RemoteWebDriver.java

```diff
@@ -927,6 +930,9 @@ public Set<Cookie> getCookies() {
 
     @Override
     public Cookie getCookieNamed(String name) {
+      if (name == null || name.isBlank()) {
```

Comment:
To fix the underlying issue, we need to add the endpoint "/session/sessionId/cookie/name" and use that to get the cookie

Reply:
I see `defineCommand(GET_COOKIE, get(cookie + "/:name"));` in `java/src/org/openqa/selenium/remote/codec/AbstractHttpCommandCodec.java`